packages/commonwealth/server/models/chain_entity.ts

```diff
@@ -32,9 +32,9 @@ export default (
 ): ChainEntityModelStatic => {
   const ChainEntity = <ChainEntityModelStatic>sequelize.define<ChainEntityInstance, ChainEntityAttributes>('ChainEntity', {
     id: { type: dataTypes.INTEGER, primaryKey: true, autoIncrement: true },
-    chain: { type: dataTypes.STRING, allowNull: false },
-    type: { type: dataTypes.STRING, allowNull: false },
-    type_id: { type: dataTypes.STRING, allowNull: false },
+    chain: { type: dataTypes.STRING, allowNull: false, unique: true },
```

Comment:
We should remove these unique: true constraints from the columns, because they don't necessarily reflect the uniqueness of the index (or we can use a unique string instead) -- unique = true means that the column itself must be unique.

Reply:
Ah your right - I had checked the database to make sure and didn't see the individual unique constraints but I guess those would only apply if you ran `.sync`?